node/router/messages/src/peer_response.rs

```diff
@@ -39,20 +39,56 @@ impl ToBytes for PeerResponse {
             return Err(io::Error::new(io::ErrorKind::InvalidInput, format!("Too many peers: {}", self.peers.len())));
         }
 
+        // A version indicator; we don't expect empty peer responses, so a zero value can serve
+        // as an indicator that this message is to be processed differently. The version value
+        // can be changed to a 2 in the future, once everyone expects it there.
+        0u8.write_le(&mut writer)?;
```

Comment:
This will require either a synchronous restart or a dedicated rollout based on a `Message::Version`.

Reply:
Ah, I didn't read the comment. We would then need to enforce that we never send 0 as the peer length

Reply:
It's not ideal, but I think this will not cause major issues, another explanatory commit is incoming